README.md

```diff
@@ -1,129 +1,108 @@
+### Dependencies
+The following lists of dependencies *must* be installed prior to building the
+application.
 
-###Dependencies Required for Building Project
-* Git 
+* Git
 * NodeJS (Current Version)
 	* [http://nodejs.org/](http://nodejs.org/ "NodeJS Site")
 * Ruby 1.9.3
-	* (Windows) During installation, Ensure to select "Add Ruby Executables to your PATH"
+  * (Windows) During installation, Ensure to select "Add Ruby Executables to
+    your PATH"
 * Java Runtime Environment > 1.7
 * Java Development Kit 7
 * Python (Latest version)
 * Ant
 	* (Windows) [https://code.google.com/p/winant/](https://code.google.com/p/winant/)
 	* (Mac/Linux/Unix) See [http://ant.apache.org/manual/install.html](http://ant.apache.org/manual/install.html)
-* Cordova 3.6.3-0.2.13 (This version addressed security issues that Google flagged and resulted in removal from the Google Play Store)
-	* Install this version by typing in the terminal 
-		* (Windows) `npm install -g cordova@3.6.3-0.2.13`
-		* (Mac/Linux/Unix) `sudo npm install -g 3.6.3-0.2.13`
+* Cordova 5.4+
+	* Install this version by typing in the terminal
+		* `[sudo] npm install -g cordova@5.4`
 * Sencha Command Line Tools 6.0.2.14 available [here](https://www.sencha.com/products/extjs/cmd-download/)
-* SASS (Ruby Gem)
-	* Type `gem install sass` after ruby has been installed
-* Compass (Ruby Gem)
-	* Type `gem install compass` after ruby has been installed
-	*
 * Android SDK (if building native Android)
-	* Ensure that you add the Android SDK to the PATH with access to the following directories:
+  * Ensure that you add the Android SDK to the PATH with access to the following
+    directories:
 		* platform-tools
 		* tools
 * iOS SDK (if building native iOS)
 
+#### Required Cordova Plugins
+Several cordova plugins are required to build the app.  Running `python
+fetchPlugins.py` from the root directory will install all dependencies.  The
+full list of requirements can be found by opening that script.
+
+### Building the Project
+1. Clone this repository and `cd` into the directory `git clone
+   https://github.com/TourTrak/devcycle-mobile-app.git && cd
+   devcycle-mobile-app`
+2. Run the command `sencha cordova init edu.rit.se.tourtrak TourTrak`.  This
+   will initialize the cordova project within the `cordova` directory.
+3. Install the required plugins as discussed in the [previous section][plugins].
+4. Execute `git submodule update --init --recursive` in order to pull down
+   [leaflet-usermarker] into the third party libraries
+5. Run `sencha app build native` build the native applications.  By default,
+   this will build both the Android and iOS applications.  If you are unable to
+   build the iOS application (i.e. not on a Mac), you must first update
+   [this line][native-build-line] to remove `ios`.  This will build the
```

Comment:
This isn't actually linking to the right place. 
Also, separate the MAC/iOS and Windows builds. He wanted to run the command before reading the entire step.